Python/pywarpx/picmi.py

```diff
@@ -1448,6 +1460,46 @@ class Simulation(picmistandard.PICMI_Simulation):
     warpx_use_filter: bool, optional
         Whether to use filtering. The default depends on the conditions.
 
+    warpx_do_multi_J: bool, default=0
+        Whether to use the multi-J algorithm, where current deposition and
+        field update are performed multiple times within each time step.
+
+    warpx_do_multi_J_n_depositions: integer
+        Number of sub-steps to use with the multi-J algorithm, when warpx_do_multi_J=1.
+        Note that this input parameter is not optional and must always be set in all
+        input files where warpx.do_multi_J=1. No default value is provided automatically.
+
+    warpx_grid_type: string, {'collocated', 'staggered', 'hybrid'}, default='staggered'
+        Whether to use a collocated grid (all fields defined at the cell nodes),
+        a staggered grid (fields defined on a Yee grid), or a hybrid grid
+        (fields and currents are interpolated back and forth between a staggered grid
+        and a nodal grid, must be used with momentum-conserving field gathering algorithm.
+
+    warpx_do_current_centering: bool, optional
+        If true, the current is deposited on a nodal grid and then centered
+        to a staggered grid (Yee grid), using finite-order interpolation.
+        Default: warpx.do_current_centering=0 with collocated or staggered grids,
+        warpx.do_current_centering=1 with hybrid grids.
+
+    warpx_field_centering_nox, warpx_field_centering_noy, warpx_field_centering_noz:
```

Comment:
This might need to be all on one line for the doc generation to be correct. Check the html. Also, the parenthesis are not needed.

Reply:
Thank you, yes, they needed to be on a single line. Note that I had to use the `/` separator, rather than `,` (comma), in order to have a single entry instead of three repeated entries. I could not find another way around it.